install_config/upgrading/manual_upgrades.adoc

```diff
@@ -1156,133 +1155,52 @@ deployed. For example, if the project is named *logging*:
-
-. Scale down your Kibana instances:
-+
-----
-$ oc scale dc/logging-kibana --replicas=0
-----
-+
-If you have an operations deployment, also run:
-+
-----
-$ oc scale dc/logging-kibana-ops --replicas=0
-----
-
-. Once confirming your Fluentd and Kibana pods have been terminated, scale down
-the Elasticsearch pods:
-+
-----
-$ oc scale dc/logging-es-<unique_name> --replicas=0
-----
-+
-If you have an operations deployment, also run:
+. Recreate the deployer templates for service accounts and running the deployer:
 +
+ifdef::openshift-enterprise[]
 ----
-$ oc scale dc/logging-es-ops-<unique_name> --replicas=0
+$ oc apply -n openshift -f \
+    /usr/share/openshift/examples/infrastructure-templates/enterprise/logging-deployer.yaml
 ----
-
-. After confirming your Elasticsearch pods have been terminated, rerun the
-deployer to generate any missing or changed features.
-
+endif::openshift-enterprise[]
 ifdef::openshift-origin[]
-.. xref:../../install_config/aggregate_logging.adoc#deploying-the-efk-stack[Re-deploy
-the EFK Stack]. After the deployer completes, re-attach the persistent volumes
-you were previously using.
-endif::openshift-origin[]
-ifdef::openshift-enterprise[]
-.. Follow the first step in
-xref:../../install_config/aggregate_logging.adoc#deploying-the-efk-stack[Deploying
-the EFK Stack]. After the deployer completes,
-re-attach the persistent volume claims you were previously using, then deploy a
-template that is created by the deployer:
-+
 ----
-$ oc process logging-support-template | oc apply -f -
+$ oc apply -n openshift -f \
+    https://raw.githubusercontent.com/openshift/origin-aggregated-logging/master/deployer/deployer.yaml
 ----
+endif::openshift-origin[]
 
-. Deployment of logging components is intended to happen automatically
-based on tags being imported into the image streams created in the previous
-step. However, as not all tags are automatically imported, this mechanism
-has become unreliable as multiple versions are released. Therefore,
-manual importing may be necessary as follows.
-+
-For each image stream `logging-auth-proxy`, `logging-kibana`,
-`logging-elasticsearch`, and `logging-fluentd`, manually import the
-tag corresponding to the `*IMAGE_VERSION*` specified (or defaulted)
-for the deployer.
-+
-----
-$ oc import-image <name>:<version> --from <prefix><name>:<tag>
-----
-+
-For example:
+. Generate any missing service accounts and roles:
 +
 ----
-$ oc import-image logging-auth-proxy:3.2.1 \
-     --from registry.access.redhat.com/openshift3/logging-auth-proxy:3.2.1
-$ oc import-image logging-kibana:3.2.1 \
-     --from registry.access.redhat.com/openshift3/logging-kibana:3.2.1
-$ oc import-image logging-elasticsearch:3.2.1 \
-     --from registry.access.redhat.com/openshift3/logging-elasticsearch:3.2.1
-$ oc import-image logging-fluentd:3.2.1 \
-     --from registry.access.redhat.com/openshift3/logging-fluentd:3.2.1
+$ oc process logging-deployer-account-template | oc apply -f -
```

Comment:
We also need to ensure that the "new" cluster role is added (unfortunately it can't go in the account template).

    $ oadm policy add-cluster-role-to-user oauth-editor \
           system:serviceaccount:logging:logging-deployer

(Use the project you created earlier (e.g., logging) when specifying this service account.)